actions/play_youtube_playlist_MOD.js

```diff
@@ -72,21 +69,32 @@ module.exports = {
     } else {
       options.bitrate = 'auto'
     }
+    if (msg) {
+      options.requester = msg.author
+    }
 
-    // Load playlist
-    const YouTube = new YTapi(`${apikey}`)
+    ytpl(url, function (err, playlist) {
+      if (err) this.displayError(data, cache, err)
```

Comment:
Either return or callNextAction
```suggestion
      if (err) return this.displayError(data, cache, err)
```